src/vm/codeman.cpp

```diff
@@ -3020,7 +3020,7 @@ void * EEJitManager::allocCodeFragmentBlock(size_t blockSize, unsigned alignment
     {
         CrstHolder ch(&m_CodeHeapCritSec);
 
-        mem = (TADDR) allocCodeRaw(&requestInfo, sizeof(TADDR), blockSize, alignment, &pCodeHeap);
+        mem = (TADDR) allocCodeRaw(&requestInfo, sizeof(CodeHeader), blockSize, alignment, &pCodeHeap);
```

Comment:
Wait - it is still does not make sense. `nUnwindInfo/unwindInfos` fields require `USE_INDIRECT_CODEHEADER` to be used. and this structure is variable size allocated out of line. The above `sizeof(TADDR)` has nothing to do with it.